### PR TITLE
Introduce namespace VectorTools

### PR DESCRIPTION
**Pull request by sebglane**

### CODE REVIEW

**Comment by sebglane** [comment]
After #131

**Comment by sebglane** [comment]
Fixes #129

**Comment by sebglane** [comment]
@j507 Could you take a look at this before I replace all project, interpolate and compute error methods by the ones defined in the new namespace? This will make a review much simpler...

**Review by j507 — CHANGES_REQUESTED** [review]
Apart from the two comments, the `VectorTools` namespace and its contents looks clean and properly programmed.

File: include/rotatingMHD/data_postprocessors.h [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+class PostprocessorVectorField : public DataPostprocessor<dim>
+{
+public:
+  PostprocessorVectorField(
+    const std::string &name,
+    const unsigned int partition = numbers::invalid_unsigned_int);
+
+  virtual void evaluate_vector_field(
+    const DataPostprocessorInputs::Vector<dim> &inputs,
+    std::vector<Vector<double>> &computed_quantities) const override;
```
[the remark this answers is not in the document]
Comment (sebglane) [reply include/rotatingMHD/data_postprocessors.h:74]: This method overloads [DataPostprocessor< dim >::evaluate_vector_field](https://dealii.org/9.2.0/doxygen/deal.II/classDataPostprocessor.html#a1ba57b598d24d64365d469a854271c68) which can compute all possible data from a vector-valued finite element solution. The name cannot be changed, but I will add some documentation.

File: include/rotatingMHD/data_postprocessors.h [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  virtual void evaluate_scalar_field(
+    const DataPostprocessorInputs::Scalar<dim> &inputs,
+    std::vector<Vector<double>> &computed_quantities) const override;
+
+  virtual std::vector<std::string> get_names() const override;
+
+  virtual std::vector<DataComponentInterpretation::DataComponentInterpretation>
+  get_data_component_interpretation() const override;
+
+  virtual UpdateFlags get_needed_update_flags() const override;
```
[the remark this answers is not in the document]
Comment (sebglane) [reply include/rotatingMHD/data_postprocessors.h:27]: Fixed in PR #141

**Comment by sebglane** [comment]
@j507 Do you think that this is ready to go? Should I run the DFG benchmark again?

**Review by j507 — APPROVED** [review]

**Pull request merged**